src/Files.App/Views/Properties/HashesPage.xaml

```diff
@@ -6,6 +6,7 @@
 	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
 	xmlns:datamodels="using:Files.App.Data.Models"
 	xmlns:helpers="using:Files.App.Helpers"
+	xmlns:interactivity="using:Microsoft.Xaml.Interactivity"
```

Comment:
This can be removed

Reply:
It may be because `OnHashInputTextChanged();` was called before `_hashInput` finished updating. I shared some suggested changes to resolve this issue.